docs/user/gcp/install_upi.md

```diff
@@ -319,15 +392,19 @@ gcloud deployment-manager deployments create ${INFRA_ID}-security --config 03_se
 ```
 
 ## Configure security variables
+Configure the variables based on the `03_security.yaml` deployment.
+If you excluded the IAM section, ensure these are set to the .email of their associated resources.
```

Comment:
`` `.email` ``

Reply:
Resolved in 657a91d789da80eb9e2e96efc4ded29f490d64a9